numerics/root_finders.hpp

```diff
@@ -35,14 +35,20 @@ Argument Brent(Function f,
 // Performs a golden-section search to find a minimum of |f| between
```

Comment:
Say "extremum" instead of "minimum", and document that it's a minimum for `<` and a maximum for `>`.  Same for Brent.